Profile hardware -  Draft Hardware Profile Submission (#947)
spdx/spdx-3-model: model/Core/Classes/Action.md

SPDX-License-Identifier: Community-Spec-1.0

# Action

> ## Summary
> Class that describes an action that has occurred .

Reviewer: ```suggestion
## Summary

Class that describes an action that has occurred.
```

Has (exactly) one blank line before and after the headings and the lists.

> ## Description
> Action define a event that has occurred. This is a Abstract Action. 

Reviewer: ```suggestion
## Description

Action defines an event that has occurred. This is an Abstract Action. 
```

> ## Metadata
> - name: Action

Reviewer: ```suggestion
## Metadata

- name: Action
```
Has (exactly) one blank line before and after the headings and the lists.

- SubclassOf: /Core/Artifact
- Instantiability: Abstract

> ## Properties
> - actionStartTime

Reviewer: ```suggestion
## Properties

- actionStartTime
```

SPDX-License-Identifier: Community-Spec-1.0

# Action

## Summary

Class that describes an action that has occurred.

## Description

Action defines an event that has occurred. This is an Abstract Action. 

## Metadata

- name: Action

> - SubclassOf: /Core/Artifact

Reviewer 2: Do not include same namespace.

- Instantiability: Abstract

## Properties

- actionStartTime

>   - type: /Core/DateTime

Reviewer 2: Do not include same namespace.

  - minCount: 0
  - maxCount: 1
- actionEndTime

>   - type: /Core/DateTime

Reviewer 2: Do not include same namespace.
